src/pages/documentation/turn/connecting.astro

```diff
@@ -0,0 +1,180 @@
+        satellite constellations from that receiver with RTCM31 output format)
+      </li>
+      <li>
+        MS-NAD83-RTCM32 (Single base solution from the closest base, determined by the system, including all available
+        satellite constellations from that receiver with RTCM32 output format)
+      </li>
+    </BulletedList>
+    <p>
+      Mountpoints that begin with MS are single base line solutions with the system automatically choosing the closest
+      base station.
+    </p>
+  </Section>
+  <Section {...page.section[3]}>
+    <p>
+      The Nevada GPS Network (formerly Washoe County GPS Network) is now being managed by the State of Utah. The Nevada
+      GPS Network provides users with real time corrections and data for post processing. The Coordinate Reference Frame
+      for this network is NAD83/94 (HARN). The Latitude, Longitude and Ellipsoidal Height of each reference station was
+      calculated by processing 24 hour files using the National Geodetic Survey (NGS) Online Positioning User Service
+      (OPUS) Projects application.
+    </p>
+    <p>
+      With an activated subscription, set your GPS unit's VRS connection address to <b>168.179.231.11</b> and <b
+        >port 2102.</b
+      > Enter your new device-specific username and password.
+    </p>
+    <h4>Connections from Source Table</h4>
+    <p>For Trimble units we have three different connections:</p>
+    <BulletedList>
+      <li>VRS_CMR works older Trimble unit</li>
+      <li>VRS_CMRp works with Trimble units that are newer but may not have the latest updates to the firmware</li>
+      <li>VRS_CMRx works with the newest up to date Trimble units</li>
+    </BulletedList>
+    <p>For non-Trimble units we have:</p>
+    <BulletedList>
+      <li>VRS_RTCM23 works older non-Trimble unit</li>
+      <li>VRS_RTCM31 works with non-Trimble units</li>
+      <li>VRS_RTCM3.2 works with non-Trimble units, gives a full GNSS solution (all available satellites)</li>
+    </BulletedList>
+    <p>
+      MS_CMR, MS_CMRP, MS_RTCM3.2 these are for connecting to single base when you are on the edge of the network or not
+      getting a strong network solution. System will automatically find the closest station and connect you.
+    </p>
+    <h4>Rinex File Downloads</h4>
+    <p>
+      The Nevada GPS network is providing 24 hour Rinex files with 5 second epoch available for download here: Nevada
+      GPS Rinex Download Due to storage limits, files can only be stored for a 90 day period, than they are permanently
+      deleted.
+    </p>
+  </Section>
+  <Section {...page.section[4]}>
+    <p>
+      If you have questions or require help contact the Support Desk from DTS at ugrc@utah.gov or 1-800-678-3440, or
+      Denise Peterson from UGRC at <a href="mailto:denisepeterson@utah.gov">denisepeterson@utah.gov</a> or 385-392-4011,
+      or Sean Fernandez from UGRC at
+      <a href="mailto:sfernandez@utah.gov">sfernandez@utah.gov</a>.
```

Comment:
**chore**: we have the [Contacts](https://github.com/agrc/gis.utah.gov/blob/main/src/components/page/Contacts.astro) component now. Please refactor to use it.